steppy/base.py

```diff
@@ -162,22 +162,29 @@ class Step:
     """
 
     def __init__(self,
-                 name,
                  transformer,
-                 experiment_directory,
+                 name=None,
+                 experiment_directory=None,
                  input_data=None,
                  input_steps=None,
                  adapter=None,
-                 is_trainable=False,
-                 cache_output=False,
-                 persist_output=False,
+
+                 is_fittable=True,
+                 force_fitting=True,
+                 cache_output=True,
```

Comment:
@kamil-kaczmarek as discusseed the default should be `cache_output=False` to prevent first time users from silent mistakes

Reply:
done